numpy/core/fromnumeric.py

```diff
@@ -2968,16 +2968,16 @@ def var(a, axis=None, dtype=None, out=None, ddof=0,
     Compute the variance along the specified axis.
 
     Returns the variance of the array elements, a measure of the spread of a
-    distribution.  The variance is computed for the flattened array by
```

Comment:
The "two spaces after a period ending sentence" are actually a PEP8 requirement, see around [here](https://www.python.org/dev/peps/pep-0008/#comments). That it is not normally enforced doesn't mean that it is a good idea to revert properly formatted code.